library/webpack-plugins/websocket-communication-plugin.js

```diff
@@ -45,7 +45,7 @@ function websocketCommunicationPlugin() {
       compiler.plugin('compilation', (compilation, { normalModuleFactory }) => {
         compilation.dependencyFactories.set(ConstDependency, new NullFactory())
         compilation.dependencyTemplates.set(ConstDependency, new ConstDependency.Template())
-        compilation.mainTemplate.plugin('require-extensions', function(source, chunk, hash) {
+        compilation.mainTemplate.plugin('require-extensions', function (source, chunk, hash) {
```

Comment:
Isn't there a rule that disallows anonymous `function` definitions? If we use them they should be given a name anyway.

Reply:
Perhaps the [`func-style`](https://eslint.org/docs/rules/func-style) in combination with the `"allowArrowFunctions": true`-setting.